data/data-mongodb/docker-compose.yml

```diff
@@ -3,4 +3,18 @@ services:
   mongo:
     image: 'mongo:5'
     ports:
-      - '27017'
+      - '27017:27017'
```

Comment:
The smoke tests shouldn't use a fix port to avoid clashes with other processes running in the host (either on CI or, more likely, locally). I realise that complicates things here as `mongosetup` then needs to reference the port from this container. Perhaps `docker-compose port` can help with that?